gradle.properties

```diff
@@ -2,11 +2,11 @@
 group=ai.h2o
 version=2.3.99999-SNAPSHOT
 # Major version of H2O release
-h2oMajorVersion=3.20.0
+#h2oMajorVersion=3.20.0
```

Comment:
TODO: before merging, ensure the gradle.properties do not use the temporary values